simularium/simularium-website
# Fix/cancel file icon

Problem
=======
A previous PR of mine left the styling incorrect on the icon that removes/cancels the uploaded file: 
    It had the default trash icon and also only appeared during hover.

There are couple little tweaks to button and select styling here, as well as some css organization.



Antd button was hard to target and change icon of, so I put `display: none` on that class and implemented a simple function on a `div` to replace it.

Old/incorrect:
![Screenshot 2023-12-14 at 2 48 04 PM](https://github.com/simularium/simularium-website/assets/24981838/6330d6e4-a122-4071-8e01-6604a3ddd737)
New:
![Screenshot 2023-12-14 at 3 27 08 PM](https://github.com/simularium/simularium-website/assets/24981838/129f2dc1-4d2e-4a1b-a951-9b84472f2d23)
Design:
![Screenshot 2023-12-14 at 2 49 11 PM](https://github.com/simularium/simularium-website/assets/24981838/a3bc0d5c-2c3f-46b7-bbf0-33db4e47f0b0)

Steps to Verify:
----------------
Should look like the [design](https://xd.adobe.com/view/5f448466-eebd-4ad2-bdd8-314a55133403-0a17/screen/4e72b109-7f4d-472e-99d5-2e0e1d53569d/specs/), and when you click the red icon, any uploaded file should be removed.
